app/Http/Requests/AddTagsApiRequest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class AddTagsApiRequest extends FormRequest
+{
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'photo_id' => 'required|exists:photos,id',
+            'litter' => 'required|array'
```

Comment:
key should be tags. Is it meant to be an array?

Reply:
It would be better to call it tags, like in the web version, but since the app uses the 'litter' key we can't change it.
We could maybe accept both 'litter' and 'tags' for some time, until all the app users have updated their apps to use 'tags'. But I guess we can do that later.
Yes, it's meant to be an array.